apps/condo/domains/property/hooks/useImporterFunctions.tsx

```diff
@@ -34,8 +38,8 @@ const createPropertyUnitsMap = (units, sections, floors) => {
         mapEditor.addSection({
             name,
             unitsOnFloor,
-            minFloor: 1,
-            maxFloor: floors,
+            minFloor,
+            maxFloor: floors + (minFloor - 1),
```

Comment:
So, for example, I have a building which is 26 floors and first living floor is 2nd

Judging by your code...

minFloor = 2
maxFloor = 26 + (2 - 1) = 27

It is not really obvious why the number of floors changes